mediator/med_phases_aofluxes_mod.F90

```diff
@@ -968,16 +983,24 @@ subroutine med_aofluxes_update(gcomp, aoflux_in, aoflux_out, rc)
     integer               , intent(out)   :: rc
     !
     ! Local variables
-    type(InternalState)      :: is_local
-    integer                  :: n                          ! indices
-    real(r8), parameter      :: qmin = 1.0e-8_r8
-    real(r8), parameter      :: p0 = 100000.0_r8           ! reference pressure in Pa
-    real(r8), parameter      :: rcp = 0.286_r8             ! gas constant of air / specific heat capacity at a constant pressure
-    real(r8), parameter      :: rdair = 287.058_r8         ! dry air gas constant in J/K/kg
-    integer                  :: maptype
-    type(ESMF_Field)         :: field_src
-    type(ESMF_Field)         :: field_dst
-    character(*),parameter   :: subName = '(med_aofluxes_update) '
+    type(InternalState)    :: is_local
+    integer                :: n                          ! indices
+    real(r8), parameter    :: qmin = 1.0e-8_r8
```

Comment:
This seems arbitrary, reason?

Reply:
I don't understand the question. What is arbitrary - the removal of subname?

Reply:
The value of `1.0e-8_r8` for `qmin`. Why is it needed? Why is that a good (general purpose) value?
The comments near the usage of this parameter mentions UFS but not the implications for other models. I'm probably just miissing something but it isn't clear to me.